examples/advanced_demo.py

```diff
@@ -365,15 +365,13 @@ def init_gui_stm32(self):
         self.tick = lvstm32.lvstm32()
         lcd.init(w=hres, h=vres)
         disp_buf1 = lv.disp_draw_buf_t()
-        buf1_1 = lcd.framebuffer(1)
-        buf1_2 = lcd.framebuffer(2)
+        buf1_1 = bytearray(hres * 50 * lv.color_t.SIZE)
```

Comment:
Why 50?

Reply:
RAM is not an issue on this board so I would normally have used a full buffer size, but my profiling from v7 development showed that buffer sizes of 1/4 to 1/5 barely decrease the rendering speed, so I picked `hres * 50` which is approximately 1/5 of the total display size.